index.yaml

```diff
@@ -1309,6 +1309,119 @@ entries:
+      catalog.cattle.io/release-name: rancher-gke-operator
+      catalog.cattle.io/scope: management
+    apiVersion: v2
+    appVersion: 1.1.1-rc1
+    created: "2021-05-21T13:39:28.33631-07:00"
+    description: A Helm chart for provisioning GKE clusters
+    digest: fcb2ffccb96a544821cd3b4e24fa17dc2c3bf1960620ac36fc42f3e80497cefd
+    home: https://github.com/rancher/gke-operator
+    name: rancher-gke-operator
+    sources:
+    - https://github.com/rancher/gke-operator
+    urls:
+    - assets/rancher-gke-operator/rancher-gke-operator-1.1.101-rc1+up1.1.1-rc1.tgz
+    version: 1.1.101-rc1+up1.1.1-rc1
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-gke-operator-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/provides-gvr: gkeclusterconfigs.gke.cattle.io/v1
+      catalog.cattle.io/rancher-version: '>= 2.6.0-alpha'
+      catalog.cattle.io/release-name: rancher-gke-operator
+      catalog.cattle.io/scope: management
+    apiVersion: v2
+    appVersion: 1.1.1-rc1
+    created: "2021-05-21T13:49:00.612945-07:00"
+    description: A Helm chart for provisioning GKE clusters
+    digest: d3944df15312c492c6e94c852fcd31e1005c8ccde64e731b8d45053b8d9c804c
+    home: https://github.com/rancher/gke-operator
+    name: rancher-gke-operator
+    sources:
+    - https://github.com/rancher/gke-operator
+    urls:
+    - assets/rancher-gke-operator/rancher-gke-operator-1.1.100-rc1+up1.1.1-rc1.tgz
+    version: 1.1.100-rc1+up1.1.1-rc1
+  rancher-gke-operator-crd:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-gke-operator-crd
+    apiVersion: v2
+    appVersion: 2.0.1-rc1
+    created: "2021-05-18T16:22:39.566946-07:00"
+    description: GKE Operator CustomResourceDefinitions
+    digest: 6a8feaa4afcbf5db22f962495c84e92747dafe1acb7ca046fa32627d808b27c1
+    name: rancher-gke-operator-crd
+    urls:
+    - assets/rancher-gke-operator-crd/rancher-gke-operator-crd-2.0.100-rc1+up2.0.1-rc1.tgz
+    version: 2.0.100-rc1+up2.0.1-rc1
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-gke-operator-crd
+    apiVersion: v2
+    appVersion: 1.1.1-rc1
+    created: "2021-05-21T13:36:21.572363-07:00"
+    description: GKE Operator CustomResourceDefinitions
+    digest: 6a4ea247a0d646da61e411874dcef9693b964bf3147069ee0dd51916dac152ea
+    name: rancher-gke-operator-crd
+    urls:
+    - assets/rancher-gke-operator-crd/rancher-gke-operator-crd-1.1.101-rc1+up1.1.1-rc1.tgz
+    version: 1.1.101-rc1+up1.1.1-rc1
```

Comment:
Don't get why we are listing 1.1.101 here either

Reply:
Removed.